### PR TITLE
General cleanup of GitHub Actions workflows.

### PR DESCRIPTION
**Pull request by Ferroin**
##### Summary

* Remove the installer workflow, it's duplicating work being done by the build workflow.
* Update the list of distros the build workflow runs against to include Debian 8 and Alpine Edge.
* Update the name of the build workflow to properly reflect that it's testing the installer.
* Update the build workflow to also check a plain build without using the installer.
* Split the `install-required-packages.sh` testing to it's own step, using a local docker image to carry the state over. This makes it easier to be certain what failed, and speeds up the other steps a bit.
* Use `netdata-all` for the `install-required-packages.sh` invocation instead of `all`. This skips installing things that we only care about for demo sites, which users are not realistically going to install.
* Re-enable CentOS 8 build checks (they were only failing because of using `all` with `install-required-packages.sh`).
* Re-enable OpenSUSE build checks (they also appear to only have been failing because of how we were invoking `install-required-packages.sh`).
* Remove checks for Fedora 29, OpenSUSE Leap 15.0, and Ubuntu 19.04. These are all EOL upstream, and we're not testing them in Travis right now either.

##### Component Name

area/ci

##### Description of testing that the developer performed

Workflwos run correctly and as expected in this PR.

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -7,10 +7,11 @@ on:
   pull_request:
 jobs:
   build:
-    name: Build
+    name: Build & Install
```
Comment (ilyam8) [inline .github/workflows/build.yml]: should you rename the file to reflect changes?

Comment (ilyam8) [reply .github/workflows/build.yml]: it is `build.yml`

Comment (Ferroin) [reply .github/workflows/build.yml]: I'm not 100% certain yet. I was kind of planning ot see what @prologic has to say about that.

Comment (prologic) [reply .github/workflows/build.yml]: Rename it :) Thats the convention I'm using.

Comment (prologic) [reply .github/workflows/build.yml]: Can we also:

```sh
mkdir .github/dockerfiles
mv .github/Dockerfile.build_test .github/dockerfiles/
```

I'd prefer it if we keep logical directory structures for our own sanity ;)

**Comment by Ferroin** [comment]
Force pushed a rebase to kick Netlify so that it hopefully reports status correctly this time...

**Pull request merged**